### ISSUE TITLE
bug(windows): modifier key occasionally is 'stuck on' 

### ISSUE THREAD

**Issue by rc-swag** [keymanapp/keyman#8064]
### Describe the bug

After #7716 addressed the serialization of modifier keys, there is still seen an occasional case where the modifier key sticks on. A log file has been obtained for further investigation.

### Reproduce the bug

Typed the sentence: `Lonh does does`
Output was: `LOnh DOes DOes`

It is likely that the release of the right shift may have been slight after pressing the `o` on the L. However `Shift` was not pressed again for the rest of the sentence so why the `DO`es

### Keyman apps

- [ ] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [ ] Keyman for macOS
- [X] Keyman for Windows
- [ ] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

15.0.275

### Operating system

Windows 11

### Additional context

@rc-swag  has the keyboards and keylogging logs. 
- [ ] Remove DebugMessages once the issue is resolved  Search for `//TODO: #8064`

**Comment by rc-swag** [comment]
Used #8033 to make a signed build with logging for each path the modifier key input can take in the low level hook.

**Comment by rc-swag** [comment]
Another @MattGyverLee has experienced the modifier key when using the IPA keyboard and setting the keyboard option to "Before". I also have some logs and am trying to find the exact sequence in the log when the key was stuck for the user versus what was intended.

**Comment by rc-swag** [comment]
Here is another [report ](https://community.software.sil.org/t/keyman-causing-windows-10-to-crash/8516)

**Comment by mcdurdin** [comment]
Another report: https://community.software.sil.org/t/8777

**Comment by rc-swag** [comment]
- [ ] add additional logging when processing a modifier key press

**Comment by rc-swag** [comment]
https://community.software.sil.org/t/vedic-sanskrit-keyboard-continuously-presses-alt-key-on-windows-11/9977

**Comment by mcdurdin** [comment]
@rc-swag notes that stuck key logs still have lowlevelkeyboardproc messages, so this probably does not resolve that issue.

**Comment by rc-swag** [comment]
Here are two different logs
I had to extract the test3_lowlevel.xslx from logs1 as the zip file was two large to upload. The two files are just to make it easier to compare filter the lowlevel messages. In the zip file is also the scenario that I think occured to make it happen and also some notes about what I have found so far.

[RC_logs_2.zip](https://github.com/user-attachments/files/23785339/RC_logs_2.zip)

[RC_Logs_1.zip](https://github.com/user-attachments/files/23785348/RC_Logs_1.zip)

[test3_lowlevel.xlsx](https://github.com/user-attachments/files/23785351/test3_lowlevel.xlsx)

**Comment by mcdurdin** [comment]
> Closed by [#15219](https://github.com/keymanapp/keyman/pull/15219).

Nope, see [above](https://github.com/keymanapp/keyman/issues/8064#issuecomment-3579778745). My bad
